README.md

```diff
@@ -18,11 +18,12 @@ Supports:
 
 These are the versions we test for:
```

Comment:
See comment in ticket regarding linking to supported versions, rather than listing.